### PR TITLE
Codegen PtrAdd and Store.

### PR DESCRIPTION
**Pull request by vext01**
Should be straightforward.

### CODE REVIEW

File: ykrt/src/compile/jitc_yk/codegen/x86_64.rs [hunk 1]
```diff
@@ -438,4 +456,34 @@ mod tests {
             &patt_lines.join("\n"),
         );
     }
+
+    #[test]
+    fn codegen_ptradd_spillalloc() {
+        let mut jit_mod = test_module();
+        //let ptr_ty_idx = jit_mod.type_idx(&jit_ir::Type::Ptr) .unwrap();
```
Comment (ltratt) [inline ykrt/src/compile/jitc_yk/codegen/x86_64.rs:467]: Commented line.

Comment (vext01) [reply ykrt/src/compile/jitc_yk/codegen/x86_64.rs]: Fixed in a1c32b15

**Comment by vext01** [comment]
OK to squash?

**Comment by ptersilie** [comment]
Please squash.

**Pull request merged**